### PR TITLE
MKon The Case For Electronic Cash

### PR DESCRIPTION
**Pull request by JOhNKmus**
@lh1008

Transcription for

The Case for Electronic Cash: Why Private Peer-to-Peer Payments are Essential to an Open Society
Jerry Brito, Coin Center

Author @smeemon

Taiga instance 205
https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/205

### CODE REVIEW

**Comment by JOhNKmus** [comment]
Starting the review on this

**Comment by netrik182** [comment]
Hi @lh1008 and @JOhNKmus. I've reviewed the "_review.md" file and it looks good. Except for having these two files, the transcription is ready to go.

File: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/The_Case_for_Electronic_Cash_review.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
...to draw that contrast, right, between what we don't want, which is something like China, and an open society, which, folks in government, especially folks in law enforcement and national security, they are buy and large doing what they do for a living because they care about liberal values. They care about the constitution and the values the constitution stands for, and they want to see an open society. There's a balance between liberty and order, and I think a lot of folks who are in law enforcement and national security and policy makers, because it's their job to keep order and sort of have more, go more toward the order side of the ledger, but I think you can make them recognize that there's a balance there, and you have to be careful that when getting order that you don't give up too much liberty. And this is why it's important to point this out to them. And what we explained is that privacy is essential to freedom of though, to freedom of speech, and to association, not only because it prevents would-be censors from discovering thought crimes, but also because of the chilling effects that come from knowing that one is being watched all of the time, especially by an authority.
```
Comment (lh1008) [inline monero-outreach-docs/en/transcriptions/monero-konferenco-2019/The_Case_for_Electronic_Cash_review.md]: they are buy...

it should be, "they are by..."

**Comment by lh1008** [comment]
Hey, @JOhNKmus great review. @netrik182 thank you so much for taking care of this. I also took a look at the transcription. There are some minor details that I was able to catch. For future reviews, years can be type as normal numbers because of being dates, e.g. twenty twenty can be 2020. 

Anyway is a great review. Thanks a lot for this.

**Comment by erciccione** [comment]
@lh1008 should this be merged?

**Pull request merged**